client/.env

```diff
@@ -0,0 +1,3 @@
+REACT_APP_API_URL=https://api.spoonacular.com/recipes
+
+REACT_APP_API_KEY=b1ffaa2fe069407e91a03674c1d497ed
```

Comment:
probably don't wanna put this here LOL 🙃🙃🙃

Reply:
It should be okay for now considering I have a "free" account and there's a limit on the number of api calls anyways. When we get access to the account with 5k requests/day I won't push that here